admin/create-dumps.sh

```diff
@@ -18,6 +18,15 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
+set -e
+
+if [[ "${CONTAINER_NAME}" == "listenbrainz-cron-prod" && "${PROD}" == "prod" ]]
```

Comment:
@mayhem You may have missed this in the merge - in f5ed0b7f0982fd54acb99230a662045ba1ab5cd2 this was moved to the top of the file and deleted from further down, but now the check also exists later in this file on line 71

Reply:
Fixed.